libslide/src/bignum.rs

```diff
@@ -0,0 +1,100 @@
+use std::fmt;
+const DIGIT_COUNT: usize = 10000;
```

Comment:
It would be cheaper to use a `Vec` IMO. I think Rust/LLVM will stack allocate small-capacity vecs.

Reply:
and just build the vec dynamically every time a new object is made?

Reply:
Do you really think its that hard? I found some c++ files where its not that bad?

Reply:
And just build the vec dynamically at runtime?

Reply:
Do they implement division and stuff? I’ve heard it’s pretty challenging. But I’d love to see it done!

Reply:
Yeah, I’ll let you make the choice on this one. Either dynamic allocation with smaller sizes or static allocation of a large number of bits. Each have their pros and cons, I’ll defer to you.

Reply:
Division is just repeated subtraction though. Once you implement that it’s done. I think the tricky think would be exp and overflows